package.json

```diff
@@ -53,9 +53,9 @@
     "chai": "^3.5.0",
     "enzyme": "^2.4.1",
     "eslint": "^3.3.1",
-    "eslint-config-airbnb": "^10.0.1",
-    "eslint-plugin-import": "^1.14.0",
-    "eslint-plugin-jsx-a11y": "^2.1.0",
+    "eslint-config-airbnb": "^14.0.0",
+    "eslint-plugin-import": "^2.2.0",
+    "eslint-plugin-jsx-a11y": "^3.0.2",
     "eslint-plugin-react": "^6.1.2",
```

Comment:
The only thing `npm ls` complains about is

> npm ERR! peer dep missing: react-native@*, required by react-native-mock@0.2.9

Which I think I might just ignore.